### PR TITLE
Added build script for apache/shiro package

### PR DESCRIPTION
**Pull request by kumarvikas2020**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [x] Did you have **Legal approvals** for patch files ? 

Package Name: shiro
Package Version: shiro-root-1.5.2
Package URL: [shiro](https://github.com/apache/shiro)

Tested on: UBI 8.4

### CODE REVIEW

File: s/shiro/shiro_1.5.2_ubi_8.4.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# ----------------------------------------------------------------------------
+#
+# Package               : shiro
+# Version               : shiro-root-1.5.2
+# Source repo           : https://github.com/apache/shiro
+# Tested on             : UBI 8.4
+# Language              : Java
+# Travis-Check          : False
```
Comment (seth-priya) [inline s/shiro/shiro_1.5.2_ubi_8.4.sh:8]: Why is Travis-Check set to False?

File: s/shiro/shiro_1.5.2_ubi_8.4.sh [hunk 2]
```diff
@@ -0,0 +1,55 @@
+# ----------------------------------------------------------------------------
+#
+# Package               : shiro
+# Version               : shiro-root-1.5.2
```
Comment (seth-priya) [inline s/shiro/shiro_1.5.2_ubi_8.4.sh:4]: is the script very specific to this version? If not, maybe we can remove the version from the script name?

File: s/shiro/shiro_1.5.2_ubi_8.4.sh (hunk 1 above)
Comment (kumarvikas2020) [reply s/shiro/shiro_1.5.2_ubi_8.4.sh]: It's been corrected.

File: s/shiro/shiro_1.5.2_ubi_8.4.sh (hunk 2 above)
Comment (kumarvikas2020) [reply s/shiro/shiro_1.5.2_ubi_8.4.sh]: Renamed as suggested.

**Review by seth-priya — APPROVED** [review]

**Pull request merged**